manifests/0000_09_service-ca-operator_06_config.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operator.openshift.io/v1
```

Comment:
EnsureOperatorConfigExists is at v1alpha1 until the bump, it seems to hate my created v1 type. After the bump we will be able to switch to doing EnsureOperatorConfigExists instead of the CVO create that we are doing now.